packages/protoplugin-example/README.md

```diff
@@ -41,7 +41,7 @@ npm test
 
 ## About this example
 
-This example is a starting point - we encourage you to try it out and experiment.
+This example is a starting point—we encourage you to try it out and experiment.
```

Comment:
Another space around dashes thing, but this seems like it was intentional, so ignore me if so.